JS_Assignment_1.js

```diff
@@ -0,0 +1,22 @@
+
+function secondLargest(array) {
+    if (array.length < 2) {
+        return null; 
+    }
+
```

Comment:
Do it with single for loop , if you hv gone through the array once  you will figure out second largest number 
 so can't the machine do it with one iteration loop ?